db/seedData.js

```diff
@@ -344,6 +387,31 @@ async function createInitialReviews(){
   }
 }
 
+async function createInitialCategories(){
+try {
+console.log("creating categories");
+const categoriestoCreate = [
+  {name: 'rock'},
+  {name: 'jazz'},
+  {name: 'metal'},
+  {name: 'country'},
+  {name: 'pop'},
+  {name: 'vinyl'},
+  {name: 'cassettes'},
+  {name: 'alternative'},
+  {name: 'rap'},
+  {name: 'R&B'}
+]
+const categories = await Promise.all(
+  categoriestoCreate.map(createCategory)
+)
+console.log("Categories", categories)
```

Comment:
remove all console.logging now that you now it works.
console.logging is for personal debugging